test/services/payments/PaymentsHistoryServiceSpec.scala

```diff
@@ -95,76 +95,76 @@ class PaymentsHistoryServiceSpec extends PlaySpec with ScalaFutures with GuiceOn
           PaymentRecord(
             reference = "reference number",
             amountInPence = 3,
-            createdOn = new DateTime("2018-10-19T08:00:00.000"),
+            createdOn = date.minusDays(1),
             taxType = "tax type"
           )
         ))
       }
 
-      "not return payment history when status is not Successful" in {
-        when(mockConnector.get(any())(any()))
-          .thenReturn(Future.successful(
-            Right(
-              List(CtPaymentRecord(
-                reference = "reference number",
-                amountInPence = 1,
-                status = Invalid,
-                createdOn = "2018-10-20T08:00:00.000",
-                taxType = "tax type")
-              )
-            )
-          ))
-
-        testService.getPayments(CtEnrolment(CtUtr("utr"), isActivated = true), date).futureValue mustBe Right(Nil)
-      }
-
-      "not return payment history when payment falls outside of 7 days" in {
-        when(mockConnector.get(any())(any()))
-          .thenReturn(Future.successful(
-            Right(
-              List(CtPaymentRecord(
-                reference = "reference number",
-                amountInPence = 1,
-                status = Successful,
-                createdOn = "2018-10-13T07:59:00.000",
-                taxType = "tax type")
-              )
-            )
-          ))
-
-        testService.getPayments(CtEnrolment(CtUtr("utr"), isActivated = true), date).futureValue mustBe Right(Nil)
-      }
-
-      "return Nil when date is invalid format" in {
-        when(mockConnector.get(any())(any()))
-          .thenReturn(Future.successful(
-            Right(
-              List(CtPaymentRecord(
-                reference = "reference number",
-                amountInPence = 1,
-                status = Successful,
-                createdOn = "invalid-date",
-                taxType = "tax type")
-              )
-            )
-          ))
-
-        testService.getPayments(CtEnrolment(CtUtr("utr"), isActivated = true), date).futureValue mustBe Right(Nil)
-      }
-
-      "return Nil when payment history could not be found" in {
-        when(mockConnector.get(any())(any()))
-          .thenReturn(Future.successful(Right(Nil)))
-
-        testService.getPayments(CtEnrolment(CtUtr("utr"), isActivated = true), date).futureValue mustBe Right(Nil)
-      }
-
-      "return Left(PaymentRecordFailure) when connector fails to parse" in {
-        when(mockConnector.get(any())(any()))
-          .thenReturn(Future.successful(Left("unable to parse data from payment api")))
-
-        testService.getPayments(CtEnrolment(CtUtr("utr"), isActivated = true), date).futureValue mustBe Left(PaymentRecordFailure)
-      }
+      //      "not return payment history when status is not Successful" in {
```

Comment:
is all this supposed to be comented out? can it be eithher deleted or uncomented?